cmake/AddConfigurationHeader.cmake

```diff
@@ -1,9 +1,16 @@
 # Generate config.h with useful configuration info.
 include(CheckFunctionExists)
-check_function_exists(mkstemp HAVE_MKSTEMP)
+check_function_exists(mkstemp MKSTEMP_FUNCTION_EXIST)
+
+include(CheckCxxSymbolExists)
```

Comment:
It should be CheckCXXSymbolExists, not CheckCxxSymbolExists. I guess it is still okay with Cygwin or MinGW because windows is case-insensitive. But this explains the travis build error.